scripts/build/file2hex.py

```diff
@@ -24,24 +24,43 @@ def parse_args():
 
     parser = argparse.ArgumentParser(
         description=__doc__,
-        formatter_class=argparse.RawDescriptionHelpFormatter, allow_abbrev=False)
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        allow_abbrev=False,
+    )
 
     parser.add_argument("-f", "--file", required=True, help="Input file")
-    parser.add_argument("-o", "--offset", type=lambda x: int(x, 0), default=0,
-                        help="Byte offset in the input file")
-    parser.add_argument("-l", "--length", type=lambda x: int(x, 0), default=-1,
-                        help="""Length in bytes to read from the input file.
-                        Defaults to reading till the end of the input file.""")
-    parser.add_argument("-m", "--format", default="list",
-                        help="Output format: 'list' (default) or 'literal' (string literal)")
-    parser.add_argument("-g", "--gzip", action="store_true",
-                        help="Compress the file using gzip before output")
-    parser.add_argument("-t", "--gzip-mtime", type=int, default=0,
-                        nargs='?', const=None,
-                        help="""mtime seconds in the gzip header.
+    parser.add_argument(
+        "-o", "--offset", type=lambda x: int(x, 0), default=0, help="Byte offset in the input file"
+    )
+    parser.add_argument(
+        "-l",
+        "--length",
+        type=lambda x: int(x, 0),
+        default=-1,
+        help="""Length in bytes to read from the input file.
+                        Defaults to reading till the end of the input file.""",
```

Comment:
`argparse` does this trick where spaces are trimmed in these multi-line `help` strings, which looks awkward when formatted now.